src/webp/vp8.rs

```diff
@@ -1421,19 +1483,19 @@ impl<R: Read> VP8Decoder<R> {
     }
 
     /// Decodes the current frame and returns a reference to it
-    pub fn decode_frame(&mut self) -> io::Result<&Frame> {
-        try!(self.read_frame_header());
+    pub fn decode_frame(&mut self) -> ImageResult<&Frame> {
```

Comment:
This is [part of the public interface](https://docs.rs/image/0.21.1/image/webp/vp8/struct.VP8Decoder.html). Need to minimally adjust that change to not change the return type here for now.